doc/nrf/ug_nrf5340.rst

```diff
@@ -113,9 +110,10 @@ Most of these samples should run on the nRF5340 PDK, but not all have been thoro
 Samples that use non-standard features of the Bluetooth Low Energy controller, like the :ref:`ble_llpm` sample, are not supported.
 Additionally, the |NCS| NFC samples are also available for nRF53 - they run only on the application core and do not require any firmware for the network core.
 
-Some samples require configuration adjustments to the :ref:`zephyr:bluetooth-hci-rpmsg-sample` sample as described in the `Network samples`_ section.
+Some BLE samples require configuration adjustments to the :ref:`zephyr:bluetooth-hci-rpmsg-sample` sample as described in the `Network samples`_ section.
 
 These samples must be programmed to the application core, in the secure domain.
+To select whether or not to enable the Network core, use :option:`CONFIG_BOARD_ENABLE_CPUNET`.
```

Comment:
You can add a sentence that this is option is selected by default when radio protocols subsystem / components are used (e.g. BLE). Or would it be too detailed?

Reply:
Added